modules/va_facilities/app/controllers/va_facilities/v0/facilities_controller.rb

```diff
@@ -68,22 +70,21 @@ def set_default_format
       private
 
       def validate_params
-        validate_bbox_lat_and_long_or_ids
+        validate_a_param_exists
         validate_bbox
         %i[lat long].each { |param| verify_float(param) } if params.key?(:lat) && params.key?(:long)
         validate_no_services_without_type
         validate_type_and_services_known unless params[:type].nil?
       end
 
-      def validate_bbox_lat_and_long_or_ids
-        bbox = params.key?(:bbox)
+      def validate_a_param_exists
+        req_params = REQUIRE_ONE_PARAM.map { |param| params.key? param }
         lat_and_long = params.key?(:lat) && params.key?(:long)
-        ids = params.key? :ids
 
-        if !bbox && !lat_and_long && !ids
-          %i[bbox long lat ids].each do |param|
+        if req_params.none? && !lat_and_long
```

Comment:
`none?` accepts a block, so you don't have to assign a `req_params` variable.
```
if !lat_and_long && REQUIRE_ONE_PARAM.none? { |param| params.key? param } &&
```
(i switched the conditions so that the less complex one can fail fast)